src/components/panel/panel.js

```diff
@@ -126,6 +126,16 @@ angular
  *   - `animation` - `{MdPanelAnimation=}`: An MdPanelAnimation object that
  *     specifies the animation of the panel. For more information, see
  *     `MdPanelAnimation`.
+ *   - `onDomAdded` - `{function=}`: Callback function used to announce when
+ *     the panel is added to the DOM. This function needs to return a promise
+ *     for chaining purposes.
+ *   - `onOpenComplete` - `{function=}`: Callback function used to announce
+ *     when the open() action is finished. This function needs to return a promise
+ *     for chaining purposes.
+ *   - `onRemoving` - `{function=}`: Callback function used to announce the
+ *     close/hide() action is starting. This allows developers to run custom
+ *     animations in parallel the close animations. This function needs to return
```

Comment:
Maybe remove the comment about custom animations. The close animation can be customized already.